### ISSUE TITLE
AssertionError in conversion to gmsh msh format

### ISSUE THREAD

**Issue by massimiliano-leoni** [nschloe/meshio#173]
Converting from some format to gmsh-msh results in the following error
```
$ meshio-convert Mesh_1.xml Mesh_1.msh
Number of points: 843838
Elements:
  Number of tetras: 4737258
WARNING:root:Binary Gmsh needs 32-bit integers (got int64). Converting.
Traceback (most recent call last):
  File "/home/mleoni/.local/bin/meshio-convert", line 135, in <module>
    _main()
  File "/home/mleoni/.local/bin/meshio-convert", line 75, in _main
    field_data=field_data
  File "/home/mleoni/.local/lib/python3.5/site-packages/meshio/helpers.py", line 171, in write
    write_binary=(file_format == 'gmsh-binary')
  File "/home/mleoni/.local/lib/python3.5/site-packages/meshio/gmsh_io.py", line 595, in write
    _write_data(fh, 'ElementData', cell_data_raw, write_binary)
  File "/home/mleoni/.local/lib/python3.5/site-packages/meshio/gmsh_io.py", line 499, in _write_data
    'Can only handle one data field right now.'
AssertionError: Can only handle one data field right now.
```

**Comment by nschloe** [comment]
Fixed via #176. Get 1.10.8. and try it out! :) (This release also contain the dolfin fix.)

**Comment by massimiliano-leoni** [comment]
Fix confirmed, thank you!
